frontend/src/app/shared/components/editor/components/ckeditor/ckeditor-setup.service.ts

```diff
@@ -75,24 +101,15 @@ export class CKEditorSetupService {
         ui: uiLocale,
         content: contentLanguage,
       },
+      link: {},
     };
 
-    return this
-      .createWatchdog(editorClass, contentWrapper, config)
-      .then((watchdog:ICKEditorWatchdog) => {
-        const { editor } = watchdog;
-        toolbarWrapper.appendChild(editor.ui.view.toolbar.element);
-
-        // Allow custom events on wrapper to set/get data for debugging
-        jQuery(wrapper)
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-return
-          .on('op:ckeditor:autosave', () => editor.config.get('autosave').save(editor))
-          .on('op:ckeditor:setData', (_, data:string) => editor.setData(data))
-          .on('op:ckeditor:clear', () => editor.setData(' '))
-          .on('op:ckeditor:getData', (_, cb:(data:string) => void) => cb(editor.getData({ trim: false })));
+    const allowedLinkProtocols = this.configurationService.allowedLinkProtocols;
+    if (allowedLinkProtocols) {
+      config.link = { allowedLinkProtocols };
+    }
```

Comment:
From [documentation](https://ckeditor.com/docs/ckeditor5/latest/features/link.html#adding-custom-protocols-in-links), the key should actually be `allowedProtocols`.

```suggestion
    if (allowedLinkProtocols) {
      config.link = { allowedProtocols: allowedLinkProtocols };
    }
```

But even with this fix I did not manage to make it work.